cmd/aperturectl/cmd/blueprints/generate.go

```diff
@@ -142,6 +148,10 @@ aperturectl blueprints generate --name=rate-limiting/base --values-file=rate-lim
 			return err
 		}
 
+		// change log.Error() to log.Info() for jsonnet trace output
+		// Note use this with std.trace in jsonnet code to get the trace output
+		log.Info().Msgf("bundle.jsonnet trace output: %s", buf.String())
```

Comment:
@karansohi 
Change the message string. The traces are coming from the entire jsonnet program, not just one file (`bundle.jsonnet`)!